docs/Job-reference.md

```diff
@@ -1733,6 +1734,53 @@ job('example') {
 
 (since 1.33)
 
+### StashBuildTrigger
+
+```groovy
+job {
+    triggers {
+        stashPullRequestTrigger {
+            cron(String cron)
+            stashHost(String stashHost)
+            username(String stashUsername)
+            password(String stashUserPassword)
+            projectCode(String projectCode)
+            repositoryName(String repositoryName)
+            ciSkipPhrases(String ciSkipPhrases)                    // optional
+            checkDestinationCommit(boolean checkDestinationCommit) // optional, default false
+            checkMergeable(boolean checkDestinationCommit)         // optional, default false
+            checkNotConflicted(boolean checkDestinationCommit)     // optional, default false
+            onlyBuildOnComment(boolean onlyBuildOnComment)         // optional, default false
+            ciBuildPhrases(String )                                // optional
+        }
+    }
+}
+```
+
+Builds pull requests from Stash. Requires the
+[Stash Pull Request Builder Plugin](https://wiki.jenkins-ci.org/display/JENKINS/Stash+pullrequest+builder+plugin).
+
```

Comment:
Add a link to the "Handling Credentials" page to give the user a warning about plain text passwords.